libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -345,7 +353,8 @@ bool ASTJsonConverter::visit(FunctionDefinition const& _node)
 		make_pair("kind", TokenTraits::toString(_node.kind())),
 		make_pair("stateMutability", stateMutabilityToString(_node.stateMutability())),
 		make_pair("visibility", Declaration::visibilityToString(_node.visibility())),
-		make_pair("overrides", _node.overrides() ? toJson(_node.overrides()->overrides()) : Json::nullValue),
+		make_pair("virtual", _node.markedVirtual()),
```

Comment:
Hm, this seems to be an unrelated change. Why not create a PR in 0.6.0 which adds virtual to the JSON exporter?

Reply:
Well, it is very related as without it we can't recreate the AST upon import. But i Can put it in another PR, sure